core/src/main/java/org/javarosa/core/io/StreamsUtil.java

```diff
@@ -7,10 +7,24 @@
 
 public class StreamsUtil {
 
-    private StreamsUtil() {
-        // private constructor
+    public static byte[] getStreamAsBytes(InputStream is) {
+        ByteArrayOutputStream baos = new ByteArrayOutputStream();
+
+        try {
+            StreamsUtil.writeFromInputToOutput(is, baos);
+            return baos.toByteArray();
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
this seems wrong to me v. just throwing an IOException as part of the method signature is there a reason we went with this?